mappings/services.json

```diff
@@ -1,4 +1,27 @@
 {
+  "settings": {
+    "index": {
+      "number_of_shards": 1
+    },
+    "analysis": {
+      "analyzer": {
+        "stemming_analyzer": {
+          "tokenizer": "standard",
+          "filter": ["standard", "lowercase", "possessive_english_stemmer", "light_english_stemmer"]
```

Comment:
https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-standard-analyzer.html
https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-standard-tokenfilter.html

The filter does nothing, but appears in all their examples so seems to be 'recommended' in case there is some future need to apply a standard filtering mechanism. On the fence as to whether to leave it in or take it out.